Mentorship Spring 2019 newspost (#1954)
ppy/osu-wiki: news/2019-01-10-community-mentorship-program-spring-2019-signups-now-open.md

---
layout: post
title: "Community Mentorship Program Spring 2019 Signups Now Open"
date: 2019-01-10 12:00:00 +0000
---

> Ever wanted to get into beatmapping for this game, but have no clue where to start at? In that case, the Community Mentorship program might be the way to start your mapping career!

Reviewer: ```suggestion
Ever wanted to get into beatmapping for this game, but have no clue where to start? In that case, the Community Mentorship program might be the way to begin your mapping career!
```
start-> begin just for variety

![](/wiki/shared/news/2019-01-10-community-mentorship-program-spring-2019-signups-now-open/banner.jpg)

> This program serves as a platform for new mappers and modders to get into the process more quickly by teaming them up with a more experienced mapper and modder in the community to show them the ropes for a total of 10 weeks.

Reviewer: ```suggestion
This program serves as a platform for new mappers and modders to get into the beatmapping process more quickly. They'll be teaming up with more experienced mappers and modders in the community that will show them the ropes for a total of ten weeks.
```
it's a bit of a run-on, and 10->ten since no comparison is being made

If you have knowledge to pass on, and the idea of mentoring someone in a more structured manner than dropping bits and pieces of knowledge into private conversations with a number of people seems appealing to you, you should totally apply for being a mentor! If you're unsure how to even map to begin with when opening the editor, and are willing to learn, you'd make a good mentee!

> **For the upcoming spring cycle we are looking for both mentors and mentees for all game modes!** 

Reviewer: ```suggestion
**For the upcoming spring cycle, we are looking for both mentors and mentees for all game modes!** 
```

The timeline is as follows (all opening and closing dates start and end at midnight UTC+0):
- Mentor signups will start on 2019-01-06 and close on 2019-01-13.
- Mentee signups will open on 2019-01-20 and close on 2019-01-30.
- The 2019 spring cycle will start on 2019-02-03 and end on 2019-04-14.

> During this time, the internal organization team will host a variety of events on the Discord server, such as mapping contests, modding puzzles, or even just things like multiplayer lobbies with integrated feedback sessions! Some mentors also hold lectures from time to time that are accompanied or followed by an open round of questions from the mentees. A compendium of those can be found over at this thread. If lectures and all aren't for you and you're more of a do-it-yourself person, this thread collects a lot of mapping puzzles that have been created within the scope of this program to mess around with yourself!

Reviewer: > A compendium of those can be found over at this thread.

what is "this thread"? needs a link
